src/packagedcode/pypi.py

```diff
@@ -303,6 +305,33 @@ def parse_setup_py(location):
     )
 
 
+def parse_requirements_txt(location):
+    """
+    Return a Package built from requirements.txt.
+    """
+    dependencies = OrderedDict()
+    with io.open(location, encoding='utf-8') as loc:
+        for req in requirements.parse(loc):
+            dependencies[req.name] = req.specs
+    package_dependencies = []
+    for dep_name, dep_specs in dependencies.items():
+        package_dependencies.append(
+            models.DependentPackage(
+                purl=PackageURL(
+                    type='pypi',
+                    name=dep_name
+                ).to_string(),
+                requirement=''.join(dep_specs[0]),
```

Comment:
Is there a case where the version is pinned with `==` ? if yes, then the version should be added to the `purl` and `is_resolved = True`